irs-api/src/main/java/org/eclipse/tractusx/irs/ess/service/BpnInvestigationJob.java

```diff
@@ -55,17 +55,12 @@ public class BpnInvestigationJob {
     private List<String> answeredNotifications;
     private JobState state;
 
-    public static BpnInvestigationJob create(final Jobs jobSnapshot, final String owner,
+    public static BpnInvestigationJob create(final Jobs jobSnapshot,
             final List<String> incidentBpns) {
-        return new BpnInvestigationJob(withOwner(jobSnapshot, owner), incidentBpns, new ArrayList<>(),
+        return new BpnInvestigationJob(jobSnapshot, incidentBpns, new ArrayList<>(),
```

Comment:
Why to add empty Arrays? Should we introduce a second Constructor which provides only firt parameters

Reply:
This is exactly what this method does. Create a BpnInvestigationJob with a given Job and incident BPNs